application/cms/views.py

```diff
@@ -1074,6 +1059,22 @@ def new_version(topic, subtopic, measure, version):
     )
 
 
+@cms_blueprint.route("/<topic>/<subtopic>/<measure>/<version>/copy", methods=["GET", "POST"])
```

Comment:
This should be a `POST`-only route, surely?

Reply:
It was suppose to be, yes.